dmn/src/main/java/io/camunda/zeebe/dmn/impl/EvaluationResult.java

```diff
@@ -8,14 +8,15 @@
 package io.camunda.zeebe.dmn.impl;
 
 import io.camunda.zeebe.dmn.DecisionResult;
+import java.util.List;
 import org.agrona.DirectBuffer;
 
 /** A successfully evaluated decision */
-public final class EvaluatedDecision implements DecisionResult {
+public final class EvaluationResult implements DecisionResult {
 
   private final DirectBuffer output;
 
-  public EvaluatedDecision(final DirectBuffer output) {
+  public EvaluationResult(final DirectBuffer output) {
```

Comment:
💭 The name of the `DecisionResult` interface that this implements was chosen instead of `EvaluationResult` because it would conflict with the `EvaluationResult` in the expression-language module. If we were to move more responsibilities into the `feel-integration` module than it's likely this conflict becomes apparent and we may need to change the names of the implementation classes. See ae4086b84957a3dee8c7f4bcb1f8504df5cd2ac4. I don't think it's a problem but just wanted to point it out.

Reply:
Thanks for pointing it out. For now, I rename the class to `DecisionEvaluationResult` instead.